2/simple_task_c++/simple_task.cpp

```diff
@@ -0,0 +1,60 @@
+#include <iostream>
+#include <vector>
+#include <ctime>
+#include <fstream>
+
+void print_nxm_mat_double(std::vector<std::vector<double>> Mat)
+{
+    for(std::vector<double> el : Mat)
+    {
+        for(double num : el)
+        {
+            std::cout << num << " ";
+        }
+        std::cout << std::endl;
+    }
+}
+
+
+int main()
+{
+
+
+    int my_size = 10000; //100000
+    int R = 5;
+    float dt = 0.001;
+    float k = 1;
+
+    std::ofstream file;
+    file.open("C:\\Users\\1\\C++\\Model\\2\\simple_task_c++\\file.txt");
```

Comment:
How would we write linux path line? Would it work on windows? I suggest checking it out.